openreview/tools.py

```diff
@@ -1457,13 +1465,10 @@ def get_profile_info(profile, n_years=None):
 
     ## Emails section
     for email in profile.content['emails']:
-        if email.startswith("****@"):
-            raise openreview.OpenReviewException("You do not have the required permissions as some emails are obfuscated. Please login with the correct account or contact support.")
         # split email
         if '@' in email:
             domain = email.split('@')[1]
             domains.add(domain)
-            emails.add(email)
```

Comment:
Since we're not using `emails` anymore, does it still need to be included in the return? Same in `get_neurips_profile_info`

Reply:
the reason I didn't want to remove it is because I didn't want to fix the API 1 scripts and tests. 

I promise I will remove it as soon as we stop giving support to API 1 venues.